### PR TITLE
add server-landscape.yml

### PR DESCRIPTION
**Pull request by schasse**

### CODE REVIEW

**Comment by schasse** [comment]
when trying to create the stack, it happens:
```
$ cc-openstack stack create -t server-landscape.yml grp17_server_landscape
Error parsing template file:///home/schasse/Documents/Uni/17-SS/cloud_computing/Cloud-computing-heat-orchestration/server-landscape.yml while parsing a block mapping
  in "<unicode string>", line 9, column 3:
      key_name: "kybranz"
      ^
expected <block end>, but found '<block mapping start>'
  in "<unicode string>", line 10, column 5:
        type: string
        ^
```
I think it's not possible to specify the values of parameters in that manner. I'd go and hardcode most of the parameters under p.e. `server1.properties`. Maybe the key can be passed via commandline.

File: server-landscape-corrected.yaml [hunk 1]
```diff
@@ -0,0 +1,145 @@
+heat_template_version: 2013-05-23
```
Comment (schasse) [inline server-landscape-corrected.yaml]: `server.yaml` has template version `2015-10-15`. Do you need the older one here?

File: server-landscape-corrected.yaml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+  server1_port:
+    type: OS::Neutron::Port
+    properties:
+      network_id: { get_param: private_net_id }
+      fixed_ips:
+        - subnet_id: { get_param: private_subnet_id }
+      security_groups: [{ get_resource: server_security_group }]
+
+  server1_floating_ip:
```
Comment (schasse) [inline server-landscape-corrected.yaml]: isn't a floating_ip association needed as well like:
```yaml
frontend_ip_assoc:
   type: OS::Neutron::FloatingIPAssociation
   properties:
     floatingip_id: { get_resource: server1_floating_ip }
     port_id: { get_resource: server1_port }
```

File: server-landscape-corrected.yaml (hunk 1 above)
Comment (KybranzF) [reply server-landscape-corrected.yaml]: I tested the **newest** heat template version of 2016-10-14 which gave me this Error:
`ERROR: The template version is invalid: "heat_template_version: 2016-10-14". "heat_template_version" should be one of: 2013-05-23, 2014-10-16, 2015-04-30, 2015-10-15, 2016-04-08`
I tested the suggested `2016-04-08` template and it works fine.

Do you need the specific `2015-10-15` version? If not, you could use the newer version too.

**Pull request merged**